File: java/negatives/f5.java
import java.util.*;
class N5 {
    static List<Integer> extractAndSort(List<Map<String,Integer>> in){
        List<Integer> out = new ArrayList<>();
        for (Map<String,Integer> m : in) if (m.containsKey("value")) out.add(m.get("value"));
        out.sort(java.util.Collections.reverseOrder());
        return out;
    }
}

File: javascript/negatives/F5.js
function extractAndSort(inputList) {
    const extracted = [];
    for (const item of inputList) {
        if ("value" in item) {
            extracted.push(item["value"]);
        }
    }
    extracted.sort((a, b) => b - a);
    return extracted;
}

module.exports = { extractAndSort };